app/serializers/item_serializer.rb

```diff
@@ -1,3 +1,9 @@
 class ItemSerializer < ActiveModel::Serializer
+  include ModelHelpers
+
   attributes :id, :name, :description, :unit_price, :merchant_id, :created_at, :updated_at
+
+  def unit_price
```

Comment:
@case-eee Thank you for the comment. The thinking behind extracting it into a `helper method` was that we use `unit_price` in multiple categories so we have a single module responsible for the conversion methods.

FYI @akintner